test/envoye2e/stackdriver_plugin/stackdriver_plugin_test.go

```diff
@@ -32,83 +32,104 @@ import (
 )
 
 const outboundStackdriverFilter = `- name: envoy.filters.http.wasm
-  config:
-    config:
-      vm_config:
-        runtime: "envoy.wasm.runtime.null"
-        code:
-          local: { inline_string: "envoy.wasm.metadata_exchange" }
-      configuration: "test"
+  typed_config:
+    "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+    type_url: envoy.extensions.filters.http.wasm.v3.Wasm
```

Comment:
not really related to this PR, but do we need to import these APIs into pilot so we can use them in EnvoyFilters?

Reply:
Eventually. They can still change.